src/arrayMethodSort.js

```diff
@@ -5,7 +5,30 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
use default value for a function parameter
```suggestion
  [].__proto__.sort2 = function(compareFunction = compareByDefault) {
```